packages/react-native-audio-api/src/plugin/withAudioAPI.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  AndroidConfig,
+  createRunOncePlugin,
+  ConfigPlugin,
+  withInfoPlist,
+  withAndroidManifest,
+} from '@expo/config-plugins';
+
+const pkg = require('react-native-audio-api/package.json');
+
+const withBackgroundAudio: ConfigPlugin = (config) => {
+  return withInfoPlist(config, (iosConfig) => {
+    if (iosConfig.modResults.UIBackgroundModes) {
+      iosConfig.modResults.UIBackgroundModes = [
+        ...Array.from(
+          new Set([...iosConfig.modResults.UIBackgroundModes, 'audio'])
+        ),
+      ];
+    } else {
+      iosConfig.modResults.UIBackgroundModes = ['audio'];
+    }
```

Comment:
```suggestion
    iosConfig.modResults.UIBackgroundModes = [
      ...Array.from(
        new Set([...(iosConfig.modResults.UIBackgroundModes ?? []), 'audio'])
      ),
    ];
```
It should give the same result, but in my option is more readable. It's up to you ;)